server/controllers/profile.js

```diff
@@ -204,4 +213,54 @@ exports.uploadImage =
       console.log(err);
       return res.status(500).send(err);
     }
-  });
\ No newline at end of file
+  });
+
+exports.manageBooking = asyncHandler(async (req, res, next) => {
```

Comment:
Since we have routes/controllers for requests, why don't we consider using them rather than defining new routes/controllers?